bootloader/src/nRF52840/usbd_dfu_mal.h

```diff
@@ -24,19 +24,23 @@
 
 #define INTERNAL_FLASH_START_ADD        0x00000000
 #define INTERNAL_FLASH_END_ADDR         0x00100000
-#define INTERNAL_FLASH_IF_STRING        "@Internal Flash   /0x00000000/1*004Ka,47*004Kg,192*004Kg,4*004Kg,4*004Kg,8*004Ka";
+#define INTERNAL_FLASH_IF_STRING        "@Internal Flash   /0x00000000/1*004Ka,47*004Kg,192*004Kg,4*004Kg,4*004Kg,8*004Ka"
+#define INTERNAL_FLASH_IF_STRING_PROT   "@Internal Flash   /0x00000000/1*004Kb,47*004Kb,192*004Kb,4*004Kb,4*004Kb,8*004Kb"
```

Comment:
A comment explaining the reason for the different letters would be helpful so that someone looking at this in the future doesn't need to reverse engineer what the letters mean and why it's important to change them. This comment can wait for the next PR.